src/motor_node.cc

```diff
@@ -341,11 +401,13 @@ int main(int argc, char* argv[]) {
     ros::Time last_sys_maint_time = last_loop_time;
     ros::Time last_joint_time = last_loop_time;
     ctrlLoopDelay.sleep();                  // Do delay to setup periodic loop delays
+    uint32_t loopIdx = 0;
 
     while (ros::ok()) {
         current_time = ros::Time::now();
         elapsed_loop_time = current_time - last_loop_time;
         last_loop_time = current_time;
+	loopIdx += 1;
```

Comment:
If the loop runs for 497 days at 50 Hz this would technically overflow I think. Not sure if that's handled or if it would even cause any issues conceptually. So it may not be a thing worth thinking about at all since 1000 things would likely go wrong until then I guess, but some people may be running their robots 24/7 in some applications so it could conceivably be reached.

Reply:
Yes but note it is 'unsigned' and also the index is only used in a modulo way so 'all is well'.